test/TestCursorSpatialProfiles.cc

```diff
@@ -100,7 +98,7 @@ class CursorSpatialProfileTest : public ::testing::Test, public ImageGenerator {
 };
 
 TEST_F(CursorSpatialProfileTest, SmallFitsProfile) {
-    auto path_string = GeneratedFitsImagePath("10 10", IMAGE_OPTS);
+    auto path_string = (TestRoot() / "data" / "images" / "fits" / "noise_10px_10px.fits");
```

Comment:
1. See comment above re: streamlining this with common functions that return the base subdirectory paths.
2. This is not a string; it's an `fs::path`. So the variable should be renamed to `path`. (This code works even though the type has changed because paths can be converted to strings implicitly.)